src/OhioBox.Moranbernate/Utils/IKeyValuePairBuilder.cs

```diff
@@ -8,22 +8,74 @@ namespace OhioBox.Moranbernate.Utils
 	public interface IKeyValuePairBuilder<T>
 	{
 		IKeyValuePairBuilder<T> Set<TValue>(Expression<Func<T, TValue>> expression, TValue value);
+		IKeyValuePairBuilder<T> Increment<TValue>(Expression<Func<T, TValue>> expression, TValue value);

```

Comment:
do you want to also add decrement syntactic sugar?

Reply:
I can implement Decrement, but it need to be an implementation since your can't neg a generic value